src/assets/javascript/app/app.js

```diff
@@ -1,3 +1,5 @@
+//Don't think this file is ever used?
```

Comment:
I agree with this. Can we double check with @hemavana where it came from?

Reply:
Yes, there might me many such file which were added as part of initial work when we were trying to use DCMAW FE code and then moved to using HMPO express code after knowing we required some field validations which was possible using the express framework. So please feel free to remove all files which arent being used.

Reply:
Removed `/assets/javascript/app/app.js `and `/assets/javascript/cookiesjs/cookies.js`